packages/overlays/inspircd.nix

```diff
@@ -0,0 +1,16 @@
+{ pkgs ? import <nixpkgs> {}, fetchFromGitHub ? pkgs.fetchFromGitHub }:
+
+{
+  nixek = import <nixos-unstable> {
```

Comment:
By using <nixos-unstable> here and <nixpkgs> above you are potentially (likely) importing 2 different versions of the nixpkgs repo which is extremely dangerous. Instead, just add to the overlays array the same way as in rsyslog.nix